data/src/main/java/me/androidbox/data/remote/event/DeleteEventWithIdRemoteUseCaseImp.kt

```diff
@@ -0,0 +1,27 @@
+package me.androidbox.data.remote.event
+
+import me.androidbox.data.remote.network.event.EventService
+import me.androidbox.data.remote.util.CheckResult.checkResult
+import me.androidbox.domain.authentication.ResponseState
+import me.androidbox.domain.event.usecase.DeleteEventWithIdRemoteUseCase
+import javax.inject.Inject
+
+class DeleteEventWithIdRemoteUseCaseImp @Inject constructor(
+    private val eventService: EventService
+) : DeleteEventWithIdRemoteUseCase {
```

Comment:
This use case is in your data layer, but should go in domain. In that case, you can't use the data related dependencies though.
The purpose of a use case is to only contain business logic, nothing related to any SDK or data related things.